Tests/UnitTests/Extensions/StringTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+// StringTests.swift
+//
+// Copyright © 2019 Button, Inc. All rights reserved. (https://usebutton.com)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+//
+	
+import XCTest
+
+class StringTests: XCTestCase {
+
+    func testIsPlainTextEmail_arbitraryString_returnsFalse() {
+        let email = "not_an_email"
```

Comment:
Maybe add a little more breadth here?

```
XCTAssertFalse("not@an_email")
XCTAssertFalse("not.an@email")
XCTAssertFalse("not.an.email")
```

Reply:
The regex being used is available here: https://emailregex.com
With a purported _99.99%_ accuracy, I don't think adding a few assertions here gets us any closer to seeing if its 100% accurate